### ISSUE TITLE
PDFSaveOptions no options available

### ISSUE THREAD

**Issue by larseberhardt** [loonghao/photoshop-python-api#37]
I try to export a PDF with PDFSaveOptions. Unfortunately I can't add any options
https://theiviaxx.github.io/photoshop-docs/Photoshop/PDFSaveOptions.html

I don't know how to add any options.

**Comment by loonghao** [comment]
Hi @larseberhardt You just need instance the `PDFSaveOptions` as an object and to set options. let me know if that can help you, cheers.

for example:
```
option = ps.PDFSaveOptions()
option.alphaChannels
option.jpegQuality
option.alphaChannels
```

a real example:
```python
import os
from tempfile import mkdtemp

from photoshop import Session

with Session() as ps:
    option = ps.PDFSaveOptions()
    option.jpegQuality = 1
    pdf = os.path.join(mkdtemp(), "test.pdf")
    ps.active_document.saveAs(pdf, option)
    os.startfile(pdf)
```

**Comment by larseberhardt** [comment]
Hi @loonghao thank you for your fast reply.

I already tried your suggestions few times but nothing happen if you set any options. I think the issue is that nothing is coded for PDFSaveOptions.
https://github.com/loonghao/photoshop-python-api/blob/master/src/photoshop/api/save_options/pdf.py

 I tried it with the win32com.client and as expected.
`
from win32com.client import Dispatch, GetActiveObject, GetObject
app = Dispatch('Photoshop.Application')
app = GetActiveObject("Photoshop.Application")
filepath = example.jpg
docRef = app.Open(filepath)
options = Dispatch('Photoshop.PDFSaveOptions')
options.PresetFile = "Example_Preset_File" 
app.ActiveDocument.SaveAs(pdf_file, options, AsCopy=False)
app.ActiveDocument.Close(Saving=2)
`

**Comment by loonghao** [comment]
Hi @larseberhardt Since the code of this API is dynamically converted using [`comtypes`,](https://pythonhosted.org/comtypes/)
This will not affect the use of the code, they are just that there is no automatic code completion function in the script editor, I will update it in a future version.

you just need to have an active document in photoshop and run the below code.
```python
import os
from tempfile import mkdtemp

from photoshop import Session

with Session() as ps:
    option = ps.PDFSaveOptions()
    option.jpegQuality = 1
    option.PresetFile = "Example_Preset_File"
    pdf = os.path.join(mkdtemp(), "test.pdf")
    ps.active_document.saveAs(pdf, option)
    os.startfile(pdf)
```

![demo](https://user-images.githubusercontent.com/13111745/93840137-e3dcaf80-fcc1-11ea-85d1-b8197d446285.gif)

**Comment by loonghao** [comment]
Hi @larseberhardt  I released a new version 0.13.0 to improve `PDFSaveOptions` and have an [example ](https://photoshop-python-api.readthedocs.io/en/master/examples.html.#save-as-pdf), Cheers.

**Comment by larseberhardt** [comment]
@loonghao thx you! It works like a charm. So handy!
